exercises/1-the-manual-menace/README.md

```diff
@@ -236,19 +242,18 @@ ansible-playbook apply.yml -e target=tools \
 
 5. Once successful; login to the cluster through the browser (using cluster URL) and navigate to the `<YOUR_NAME>-ci-cd`. You should see Nexus up and running. You can login with default credentials (admin / admin123) ![nexus-up-and-running](../images/exercise1/nexus-up-and-running.png)
 
-
-<!-- #### 3 - Commit CI/CD -->
-
 ### Part 3 - Commit CI/CD
 
 1. Navigate to GitLab login page. You can login using your cluster credentials using the LDAP tab
 ![gitlab-ui](../images/exercise1/gitlab-ui.png)
 
 2. Once logged in create a new project called `enablement-ci-cd` and mark it as internal. Once created; copy out the `git url` for use on the next step.
 ![gitlab-new-project](../images/exercise1/gitlab-new-project.png)
-<!-- <p class="tip">
-Note - we would not normally make the project under your name but create a group and add the project there on residency but for simplicity of the exercise we'll do that here
-</p> -->
+<!-- 
```

Comment:
this note/tip is not shown, so maybe just remove it all together? @springdo thoughts?